stdlib/all_jlls.version

```diff
@@ -0,0 +1,28 @@
+## All `_jll` names and versions
```

Comment:
Is it possible to do something automatic here - like `include *.version` so that we don't have to update manually here in the future?

Reply:
It looks like you can just use wildcards.
Perhaps open a new pr?

```
include foo *.mk $(bar)
```

[Include (GNU make)](https://www.gnu.org/software/make/manual/html_node/Include.html)

Reply:
Sure.